test/winston-mongodb-test.js

```diff
@@ -14,6 +14,8 @@ const MongoDB = require('../lib/winston-mongodb').MongoDB;
 
 const dbUrl = process.env.USER_WINSTON_MONGODB_URL
     ||process.env.WINSTON_MONGODB_URL||'mongodb://localhost:27017/winston';
+const dbName = process.env.WINSTON_MONGODB_DBNAME
+    ||'otherWinston';
```

Comment:
What's the reasoning behind 'otherWinston' here?

Reply:
Just an example database name for the test. So checking the results there should be two databases logged to 'winston' and 'otherWinston'.